backend/application/advertisement.ts

```diff
@@ -82,4 +78,19 @@ export class AdvertisementApplicationService {
       )
     }
   }
+
+  async deleteAdvertisement(id: string) {
+    try {
+      // deleted_atに値を渡して論理削除
+      const deleteAdvertisement = await prisma.advertisement.update({
+        where: { id },
+        data: { deleted_at: new Date() },
+      })
+      return deleteAdvertisement
```

Comment:
```suggestion
      const deletedAdvertisement = await prisma.advertisement.update({
        where: { id },
        data: { deleted_at: new Date() },
      })
      return deletedAdvertisement
```

updateの時と同じなんで注意です。
今の状態を宣言している定数にはいれましょう！